src/main/java/duke/storage/TaskList.java

```diff
@@ -0,0 +1,122 @@
+package duke.storage;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import duke.DukeException;
+import duke.task.Task;
+
+/**
+ * Represents a list of tasks
+ */
+public class TaskList {
+    private static TaskList instance = null;
```

Comment:
What does instance refer to here? Perhaps another variable name would be more descriptive